Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/WOfSImputation.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>request adding geohash to the tags index!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>We don't use geohash directly here, unlike in some notebooks, so I'm a bit reluctant to do so</p>

Reply:
Was thinking if I was searching for the utilisation of the search by geohash function, this would be a useful use case. Happy to not have it if you think it's best.

Reply:
I think that would end up with every waterbodies notebook also tagged with geohash. Maybe a good thing? Unsure.